{"timestamp":1546337429413,"status":409,"error":"Conflict","exception":"org.apache.fineract.cn.lang.ServiceException","message":"Account Account4 already exists.","path":"/accounting/v1/accounts"}
